### PR TITLE
Users/nigurr/change gradle plugin info

### PR DESCRIPTION
**Pull request by nigurr**
Class Directories are not required to be given as input. Gradle plugin will figure out the right one.

### CODE REVIEW

File: Tasks/Gradle/task.json [hunk 1]
```diff
@@ -117,8 +117,8 @@
             "defaultValue": "build/classes/main/",
             "required": true,
             "groupName": "codeCoverage",
-            "helpMarkDown": "Comma separated list of directories containing class files, archive files(jar, war etc.). Directories and archives are searched recursively for class files. Code coverage is reported for class files present in the directories. For example, classes1/Tests,classes2/Tests2. Refer https://docs.gradle.org/current/userguide/jacoco_plugin.html.",
-            "visibleRule": "codeCoverageTool != None"
+            "helpMarkDown": "Comma separated list of directories containing class files, archive files(jar, war etc.). Directories and archives are searched recursively for class files. Code coverage is reported for class files present in the directories. By default it will be recursively searching for classes under build/main/classes which is default class directory for gradle builds",
```
Comment (allendm-msft) [inline Tasks/Gradle/task.json]: "Comma separated list of directories containing class files, archive files(jar, war etc.). Directories and archives are searched recursively for class files. Code coverage is reported for class files present in the directories. The task searches for classes under build/main/classes which is default class directory for gradle builds"

File: Tasks/Gradle/task.json [hunk 2]
```diff
@@ -117,8 +117,8 @@
...
+            "visibleRule": false
```
Comment (allendm-msft) [inline Tasks/Gradle/task.json]: if this is not going to be visible at all then how will updating the help text help ?

Comment (allendm-msft) [inline Tasks/Gradle/task.json]: is ps1 handling this properly when the node is not visible/present?

**Comment by allendm-msft** [comment]
share what testing has been done for approval

**Comment by nigurr** [comment]
E2E Testing along with Manual testing done.

**Comment by AshwiniChalla** [comment]
Approved

**Pull request merged**